### PR TITLE
Integrated with travis tests

### PR DESCRIPTION
**Pull request by pepicrft**
Project's been integrated with travis tests. I've added .travis.yml to execute automatically tests on travis
@luisrecuenco  to enable tests you've to sign up on Travis and enable this repo. This way every time  someone uploads a new PR travis will execute the build and test and will give an :+1: indicating can be merged without problems.

### CODE REVIEW

File: Example/LRScrollingSidebarControllerExampleTests/LRScrollingSidebarControllerExampleTests.m [hunk 1]
```diff
@@ -28,7 +28,6 @@ - (void)tearDown
 
 - (void)testExample
```
Comment (luisrecuenco) [inline Example/LRScrollingSidebarControllerExampleTests/LRScrollingSidebarControllerExampleTests.m]: Remove this method completely, it doesn't make much sense to have it around

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: objective-c
+xcode_sdk: iphonesimulator
+rvm: 1.9.3
+
+before_install:
+  - cd Example
+  - brew update
+script:
+  - xctool -project LRScrollingSidebarControllerExample.xcodeproj -scheme LRScrollingSidebarControllerExample -configuration Debug -arch i386 test -sdk iphonesimulator ONLY_ACTIVE_ARCH=NO
```
Comment (luisrecuenco) [inline .travis.yml:11]: Standard version of xctool has problems when launching simulator. Use this better:

brew update && brew uninstall xctool && brew install xctool --HEAD

Comment (pepicrft) [inline .travis.yml:11]: Modified :+1: @luisrecuenco

**Comment by pepicrft** [comment]
Reviewed your comments @luisrecuenco

**Comment by pepicrft** [comment]
Ping @luisrecuenco enable it and we can check if it works :smile:

**Pull request merged**